src/popover/index.js

```diff
@@ -78,7 +78,7 @@ class Popover extends Component {
   }
 
   _getBackDropElement = () => {
-    if (!this.props.onClickOut) return null;
+    if (!this.props.onClickOut || !this.props.isOpen) return null;
```

Comment:
yeah negative or guards can be confusing

maybe just

if (clickout && isOpen) return (
  stuff
);

??